src/controllers/AssetInfoController.js

```diff
@@ -142,6 +155,7 @@ export class AssetInfoController {
         },
       },
     };
+    this.usdPriceAssets = undefined;
```

Comment:
I guess naming it `usdPrices` makes more sense. Right now it makes me think that it contains assets representing usd prices for us. But that doesn't seem to be true. Plus the method `updateUsdPriceAssets` better be named `updateUsdPrices`. Or do I miss something?